src/main/kotlin/no/nav/syfo/application/service/VurderingService.kt

```diff
@@ -59,12 +59,10 @@ class VurderingService(
             )
         }
 
-        val pdf = if (vurdering.shouldJournalfores()) {
-            vurderingPdfService.createVurderingPdf(
-                vurdering = vurdering,
-                callId = callId,
-            )
-        } else null
+        val pdf = vurderingPdfService.createVurderingPdf(
+            vurdering = vurdering,
+            callId = callId,
+        )
```

Comment:
Tenkte det var like greit å bare fjerne hele den logikken med `shouldJournalfores`? Gir oss litt fordeler ved at vi alltid har en pdf også.

Reply:
Siden vi har exhaustive when på `VurderingType` for disse journalførings-greiene så er det OK :+1: